src/sharedInterfaces/queryResult.ts

```diff
@@ -55,9 +55,7 @@ export interface QueryResultWebviewState extends ExecutionPlanWebviewState {
     tabStates?: QueryResultTabStates;
     isExecutionPlan?: boolean;
     selection?: ISlickRange[];
-    executionPlanState: ExecutionPlanState & {
-        xmlPlans?: string[];
-    };
+    executionPlanState: ExecutionPlanState;
```

Comment:
Probably a good followup task will be to add js doc comments to all the interfaces and properties.

Reply:
i added some for now on the less self-explanatory ones